libraries/AP_GPS/AP_GPS_SBF.h

```diff
@@ -163,11 +180,68 @@ class AP_GPS_SBF : public AP_GPS_Backend
         float Cov_VuDt;
     };
 
+    struct PACKED msg4226 // INSNavGeod
+    {
+        uint32_t TOW;
+        uint16_t WNc;
+        uint8_t Mode;
+        uint8_t Error;
+        uint16_t Info;
+        uint16_t GNSSAge;
+        double Latitude;
+        double Longitude;
+        double Height;
+        float Undulation;
+        uint16_t Accuracy;
+        uint16_t Latency;
+        uint8_t Datum;
+        uint8_t Reserved;
+        uint16_t SBList;
+        // INSNavGeodPosStdDev sub-block
```

Comment:
Are the sub blocks guaranteed to always be sent? I couldn't find that either way in the manual.

Reply:
No the user is able to disable selected sub-block. With the command "setINSNavConfig" Therefore it is important to advice the user not setting this command. All sub-blocks are on by default and should be.

![image](https://user-images.githubusercontent.com/42541548/65863146-df600e00-e36f-11e9-9376-c2d392cc8132.png)

Reply:
There are the bits to decode what subblocks are present though, so why not use them to decode what we actually need. And if we really need all the subblocks on we should be sending that in the config strings. (I'd even support just turning on the subblocks we need).

Reply:
I would assume for now we only need the velocity sub-block and possibly the attitude sub-block? For accuracy the StdDev sub-blocks can be used but now there is no implementation for it